fs.go

```diff
@@ -0,0 +1,116 @@
+package checkup
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const indexName = "index.json"
+
+// FS is a way to store checkup results on the local filesystem.
+type FS struct {
+	// The path to the directory where check files will be stored.
+	Dir string `json:"dir"`
+	// The URL corresponding to fs.Dir.
+	Url string `json:"url"`
```

Comment:
This should be `URL` according to https://github.com/golang/go/wiki/CodeReviewComments#initialisms.